media/css/firefox/sync.less

```diff
@@ -95,17 +96,25 @@
         padding-bottom: @baseLine * 2;
     }
     .buttons {
-        .span(8);
-        .offset(2);
+        .span(10);
+        .offset(1);
+        float: none;
         clear: both;
 
         > div {
             padding: 0 (@baseLine * 5) @baseLine;
         }
 
-        > div.show-fx-31-signed-in {
+        > div.show-fx-31-signed-in,
+        > div.show-fx-31-signed-out {
             padding: 0 @baseLine @baseLine;
         }
+
+        > .show-fx-31-signed-in p {
+            .font-size(1.125rem);
```

Comment:
This is a nit, but our code is starting to jump around from using `px` to `rem` everywhere. I don't mind what unit we use, but I think we should remain consistent for our own sanity?